presto-main/src/main/java/io/prestosql/metadata/ProcedureRegistry.java

```diff
@@ -88,7 +89,7 @@ public Procedure resolve(CatalogName catalogName, SchemaTableName name)
     private void validateProcedure(Procedure procedure)
     {
         List<Class<?>> parameters = procedure.getMethodHandle().type().parameterList().stream()
-                .filter(type -> !ConnectorSession.class.isAssignableFrom(type))
```

Comment:
We can do this as two filters, which makes it a bit easier to read and modify later
```java
.filter(type -> !ConnectorSession.class.isAssignableFrom(type))
.filter(type -> !ConnectorAccessControl.class.isAssignableFrom(type))
```